### PR TITLE
Includes language in reports (closes #22).

### LINKED ISSUE #22: Reports should include user language.
Ideally, this will be `navigator.language` from the reported page.

### PR DESCRIPTION
**Pull request by chuckharmston**

### CODE REVIEW

File: src/data/language.js [hunk 1]
```diff
@@ -0,0 +1 @@
+self.port.emit('language', navigator.language);
```
Comment (pdehaan) [inline src/data/language.js:1]: Do we only want the primary language? or the full array of `navigator.languages`?

Comment (chuckharmston) [reply src/data/language.js:1]: Just the primary language.

**Comment by chuckharmston** [comment]
Updated to accommodate 7b5e503.

**Review by lmorchard — APPROVED** [review]
:+1: My one thought is if there's potential these separate content scripts will pile up & have a negative performance impact through the overhead - not the scripts themselves, because they're tiny. But, I have no specific reason to think so and it's not like there'll be hundreds of them. So :man_shrugging:

**Comment by chuckharmston** [comment]
> My one thought is if there's potential these separate content scripts will pile up & have a negative performance impact through the overhead

There may be a minor penalty here, but I'm building this as pluggably as possible for future refactoring into a package that will allow similar data collection Test Pilot experiments.

**Pull request merged**